microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/core/ItemMetadataReader.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.microprofile.jdt.core;
+
+import com.google.gson.Gson;
+import com.google.gson.stream.JsonReader;
+import com.redhat.microprofile.commons.metadata.ItemMetadata;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.StandardCharsets;
+
+public class ItemMetadataReader {
```

Comment:
please remove this class, the load of JSON must be done in AbstractStaticPropertiesProvider. If we see that the JSON load must be done in an another code, we will move this load into an utilitilies class. The idea is to try to limit utilities class. More you will have utilities class, more it will be hard for contributor to understand which utility it can use.